src/routes/docs/references/[version]/[platform]/[service]/(components)/RateLimits.svelte

```diff
@@ -2,6 +2,7 @@
     import type { SDKMethod } from '$lib/utils/specs';
 
     export let method: SDKMethod;
+    export let platformType: String;
```

Comment:
We use lowercase `string` here :+1: 
```suggestion
    export let platformType: string;
```